lib/main.dart

```diff
@@ -30,11 +32,45 @@ class MyHomePage extends StatefulWidget {
 }
 
 class _MyHomePageState extends State<MyHomePage> {
-  int _counter = 0;
+  late Uri _url;
+
+  Future<void> _launchUrl() async {
+    if (!await launchUrl(_url)) {
+      throw 'Could not launch $_url';
+    }
+  }
+
+  @override
+  void initState() {
+    super.initState();
+    initPlatformState();
+  }
+
+  Future<void> initPlatformState() async {
+    String? alpha2;
+    String? fluentelPhoneNumber;
+
+    try {
+      alpha2 = await FlutterSimCountryCode.simCountryCode;
+    } on PlatformException {
+      alpha2 = 'Failed to get sim country code.';
+    }
```

Comment:
I thought the sim country code would be the calling code number, but it's the alpha2. Which is good because Canada and Puerto Rico are also +1. But I'm guessing the SIM card country code for each would be CA and PR. This would let us expand to these places with a more focused number local to them.